backend/src/main/java/com/crm/config/SecurityConfiguration.java

```diff
@@ -26,17 +26,25 @@
     private final JwtUserDetailsService userDetailsService;
     private final JwtRequestFilter jwtRequestFilter;
 
+    private static final String[] AUTH_WHITELIST = {
+            // -- swagger ui
+            "/v2/api-docs",
+            "/swagger-resources/**",
+            "/configuration/ui",
```

Comment:
I think, that moste of the links are not  necessary

Reply:
Corrected. Some of the links had to stay because otherwise Swagger does not work